habitat/tasks/vln/vln.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
```

Comment:
You can include your names as owners/maintainers of the VLN task here as well as reflect your contribution in README section.

Reply:
I added our names in this file. For the README section, we can wait until we have a paper (as per discussion).